test/harness/wasm-module-builder.js

```diff
@@ -155,9 +169,11 @@ class WasmModuleBuilder {
     this.imports = [];
     this.exports = [];
     this.globals = [];
+    this.exceptions = [];
```

Comment:
I don't think we should push code concerned with proposed extensions into spec master.

Reply:
This PR would bring the wasm-module-builder back in sync with the V8 version of it. I can clean up the wasm-module-builder in V8 and do the sync afterwards, but I cannot remove the support of exceptions because we need it to test exceptions in V8.
If you say that there is no value in keeping the two versions in sync I will just close this PR.

Reply:
Well, I think there *is* value in keeping them in sync, but only up to experimental extensions. For reasons of process, those shouldn't be in the spec repo, even though I understand that that makes it harder to merge desirable changes.